test/e2e/specs/specs-playwright/wp-plans_purchases-spec.ts

```diff
@@ -0,0 +1,116 @@
+import {
+	DataHelper,
+	LoginFlow,
+	setupHooks,
+	SidebarComponent,
+	PlansPage,
+	IndividualPurchasePage,
+	CartCheckoutPage,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+/**
+ * Constants
+ */
+
+const userOnPremiumPlan = 'defaultUser';
+
+describe( DataHelper.createSuiteTitle( 'Plans: Purchases' ), function () {
+	let page: Page;
+	let plansPage: PlansPage;
+	let purchasesPage: IndividualPurchasePage;
+	let cartCheckoutPage: CartCheckoutPage;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe( 'Initial navigation', function () {
+		it( 'Log In', async function () {
+			const loginFlow = new LoginFlow( page, userOnPremiumPlan );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Navigate to Plans from sidebar', async function () {
+			const sidebarComponent = new SidebarComponent( page );
+			await sidebarComponent.gotoMenu( { item: 'Upgrades', subitem: 'Plans' } );
+		} );
+
+		it( 'Land on the "My Plan" tab', async function () {
+			plansPage = new PlansPage( page );
+			await plansPage.validateActiveNavigationTab( 'My Plan' );
```

Comment:
Personally, I would prefer to explicitly perform actions in POMs, even if the action may not be necessary (like in this case), for a number of reasons:

- guards against situation where default landing tab in Upgrades >> Plans change from 'My Plan' to something else in the future.
- the e2e spec reads more like  series of commands that _should work_

Would you be open to something like this?

```
// in PlansPage

async clickTab( name: PlansPageTab ) {
    // click on tab
    // check that tab is indeed clicked
}
```

An example of this can be found in [MarketingPage](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/lib/pages/marketing-page.ts#L28) and [MediaPage](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/lib/pages/media-page.ts#L86).

Reply:
Sure, that makes a lot of sense! I updated the start of the test to reflect that, and left it in for the test for the automatic navigation after cart removal.